### PR TITLE
Feat/manager 7389: Anthos dashboard IPs tab

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/anthos`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7389
| License          | BSD 3-Clause

## Description

Anthos dashboard IPs tab

### CODE REVIEW

File: packages/manager/modules/anthos/src/anthos-tenants.service.js [hunk 1]
```diff
@@ -52,11 +52,21 @@ export default class AnthosTenantsService {
   }
 
   getTenantPublicIPs(serviceName) {
-    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/public`);
+    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/public`, {
+      headers: {
+        Pragma: 'no-cache',
+        'X-Pagination-Mode': 'CachedObjectList-Pages',
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/anthos-tenants.service.js:58]: By default, only 10 items are fetched. I notice that no page size is specified so only the first 10 items will be returned each time

File: packages/manager/modules/anthos/src/dashboard/ips/template.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
+<div>
+
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/ips/template.html:2]: Needed space ?

File: packages/manager/modules/anthos/src/dashboard/ips/template.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                        data-ng-disabled="$row.inUse"
+                    >
+                        <span class="oui-icon oui-icon-trash"></span>
+                    </button>
+                </oui-datagrid-column>
+            </oui-datagrid>
+        </div>
+    </div>
+
+
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/ips/template.html:122]:
```suggestion
```

File: packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "anthos_dashboard_ips_title": "Addresses IP",
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json:2]:
```suggestion
  "anthos_dashboard_ips_title": "Addresses IPs",
```
Not sure though

File: packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "anthos_dashboard_ips_description": "Dans la console Anthos, vous avez besoin d’allouer des IPs à chacun de vos déploiements Anthos. Vous pouvez assigner autant d’IPs/ranges d’IPs que vous voudrez à votre déploiement Anthos. Vous bénéficiez également de 50 IPs publiques et pouvez en commander de nouvelles pour un faible tarif fixe.",
+  "anthos_dashboard_ips_datagrid_public_actions_menu": "Actions",
+  "anthos_dashboard_ips_datagrid_public_actions_order_ip": "Commander des IP publiques",
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json:5]:
```suggestion
  "anthos_dashboard_ips_datagrid_public_actions_order_ip": "Commander des IPs publiques",
```
sometimes we pluralize, sometimes we don't. I'm not sure what the rule is but we should be consistent across all content

File: packages/manager/modules/anthos/src/anthos-tenants.service.js [hunk 6]
```diff
@@ -8,8 +8,8 @@ export default class AnthosTenantsService {
     this.ovhDocUrl = ovhDocUrl;
   }
 
-  fetch(endPoint) {
-    return this.$http.get(endPoint).then(({ data }) => data);
+  fetch(endPoint, headers = undefined) {
```
Comment (frenautvh) [inline packages/manager/modules/anthos/src/anthos-tenants.service.js:11]: what is the point of providing "undefined" as a default value, won't it be undefined by default if not provided ?

File: packages/manager/modules/anthos/src/anthos-tenants.service.js [hunk 7]
```diff
@@ -52,11 +52,21 @@ export default class AnthosTenantsService {
...
+      },
+    });
   }
 
   getTenantPrivateIPs(serviceName) {
-    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/private`);
+    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/private`, {
+      headers: {
+        Pragma: 'no-cache',
+        'X-Pagination-Mode': 'CachedObjectList-Pages',
```
Comment (radireddy) [inline packages/manager/modules/anthos/src/anthos-tenants.service.js:67]: above commend from Marie applies here too.

File: packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json (hunk 4 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json:2]: Yes better, in others content they use IPs.
So for consistency i will use IPs.

Thanks

File: packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json (hunk 5 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json:5]: Same as here:
https://github.com/ovh/manager/pull/5456#discussion_r699095040

File: packages/manager/modules/anthos/src/dashboard/ips/template.html (hunk 3 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/dashboard/ips/template.html:122]: Removed extra space.

File: packages/manager/modules/anthos/src/anthos-tenants.service.js (hunk 6 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/anthos-tenants.service.js:11]: True, not required.

File: packages/manager/modules/anthos/src/anthos-tenants.service.js (hunk 7 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/anthos-tenants.service.js:67]: Taken in account.

File: packages/manager/modules/anthos/src/dashboard/ips/template.html (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/dashboard/ips/template.html:2]: For visibility, it's better to have a blank line.
Or may be you discuss on another aspect ?

**Review by radireddy — APPROVED** [review]

**Review by varun257 — APPROVED** [review]

**Pull request merged**